x-pack/docs/en/security/securing-communications/update-tls-certificates.asciidoc

```diff
@@ -0,0 +1,529 @@
+[[update-node-certs]]
+== Updating node security certificates
+If your cluster certificates expired or you have a new Certificate Authority
+(CA), then you'll need to update the security certificates for both the
+transport and HTTP layers. When updating your node security certificates, there
+are two main scenarios that you'll encounter:
+
+* <<update-node-certs-same,Your certificate authority (CA) or key remains the same>>
+* <<update-node-certs-different,Your CA certificate or key changes>>
+
+Regardless of the scenario, {es} monitors the SSL resources for updates
+on a five-second interval. You can just copy the new certificate and key files
+(or keystore) into the {es} configuration directory and your nodes will detect
+the changes. 
+
+Because {es} doesn't reload the `elasticsearch.yml` configuration,
+you must use *the same file names* so that {es} refreshes the security
+certificates and keys. This method of updating certificates is preferred
+because it doesn't require a full cluster restart.
+
+If you need to update the `elasticsearch.yml` configuration or change
+certificate passwords that are stored in the
```

Comment:
certificates don't have passwords, keys and keystores usually do

Reply:
++ I'll change to read:

>If you need to update the `elasticsearch.yml` configuration or change passwords for keys or keystores that are stored in the <<secure-settings,secure settings>>..."